PowerEditor/installer/nativeLang/hongKongCantonese.xml

```diff
@@ -1118,6 +1118,7 @@
                     <Item id="6903" name="搵完嘢，保持開著個對話框"/>
                     <Item id="6904" name="喺所有打開咗嘅文件執行取代之前，要先做確認"/>
                     <Item id="6905" name="取代完唔好跳去另一個關鍵字"/> <!-- #new v7.9.6 -->
+                    <Item id="6906" name="搜尋結果視窗：一行顯示唔重複（未支援右至左文字模式）"/> <!-- #new v8.4.3 -->
```

Comment:
"(not applied to RTL mode)" has been removed in this entry, see: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/059bca3315dbd71fbf235c8bc87eaacbd87a3cec
Sorry for the inconvenience.

Reply:
Has push an update. Thanks!